### PR TITLE
some refactoring on css and package.json

### PR DESCRIPTION
**Pull request by hjlee0704**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules
+bundle-3003.js
+bundle-3003.js.map
```
Comment (S1hane) [inline .dockerignore]: I don't think we need bundle-3003 in here anymore?

We could probably add bundle-review.js*

Comment (S1hane) [reply .dockerignore]: Also, let's add:

.DS_Store

Comment (hjlee0704) [reply .dockerignore]: oh yea thanx for checking 
i will modify this file as well!

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+version: '3'
+services:
+  app:
+    container_name: review
+    restart: on-failure
+    build: .
+    ports:
+      - '80:3003'
```
Comment (S1hane) [inline docker-compose.yml]: How come we need port 80 here?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+
+EXPOSE 3003
+
+CMD ["npm","run","server"]
```
Comment (S1hane) [inline Dockerfile]: I think we will need to update this to npm start

File: package.json [hunk 4]
```diff
@@ -5,14 +5,14 @@
...
-    "server": "nodemon backend/index.js",
-    "client": "webpack --watch -d source-map",
+    "start": "node backend/index.js",
+    "build": "webpack --watch",
+    "build-dev": "webpack --watch -d source-map",
     "seed": "node database/seed.js"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
```
Comment (S1hane) [inline package.json:15]: Do we need react and react-dom in here? How about axios? I think some dependencies are missing. 
Also, babel runtime can go to dev.

**Review by S1hane — CHANGES_REQUESTED** [review]

File: .dockerignore (hunk 1 above)
Comment (S1hane) [reply .dockerignore]: you can remove the .DS_Store file that was added with:
git rm

Comment (S1hane) [reply .dockerignore]: Also the bundles, can we remove those with git rm too?

File: docker-compose.yml (hunk 2 above)
Comment (hjlee0704) [reply docker-compose.yml]: oh my 3003 was taken at that time from something so i used 80 for that time i will change back to 3003

File: Dockerfile (hunk 3 above)
Comment (hjlee0704) [reply Dockerfile]: yes

**Pull request closed without merging**